pkg/yurthub/proxy/local/local.go

```diff
@@ -152,14 +159,13 @@ func (lp *LocalProxy) localWatch(w http.ResponseWriter, req *http.Request) error
 	if !ok {
 		err := fmt.Errorf("unable to start watch - can't get http.Flusher: %#v", w)
 		util.Err(errors.NewInternalError(err), w, req)
-		return err
+		return nil
```

Comment:
I understand the intention of `return nil` but the logic looks counterintuitive. 

How about call `util.Err()` in every failure case in localWatch/localPost/localReqCache and remove the code
`util.Err(errors.NewBadRequest(err.Error()), w, req)` in line 86?  The refactor effort should be small.

Reply:
fixed

Reply:
Thanks for addressing the comment.